### PR TITLE
Update node to v0.12.2

### PR DESCRIPTION
**Pull request by wind-rider**
This is a security & bugfix update of the existing v0.12 package

### CODE REVIEW

**Comment by wind-rider** [comment]
Please have a look at this pull request.

File: cross/node/digests [hunk 1]
```diff
@@ -1,3 +1 @@
-node-v0.12.0.tar.gz SHA1 d08810034d170c19759cb38bfc9442ab6b0ebc7a
-node-v0.12.0.tar.gz SHA256 9700e23af4e9b3643af48cef5f2ad20a1331ff531a12154eef2bfb0bb1682e32
-node-v0.12.0.tar.gz MD5 62c8d9c74c8f68193f60e4cba020eb48
+node-v0.12.2.tar.gz SHA256 ac7e78ade93e633e7ed628532bb8e650caba0c9c33af33581957f3382e2a772d
```
Comment (Diaoul) [inline cross/node/digests]: Did you use make digests to generate this? Why only one?

**Comment by Diaoul** [comment]
LGTM

File: cross/node/digests (hunk 1 above)
Comment (wind-rider) [inline cross/node/digests]: I didn't know the 'make digests' buildstep and one seemed good enough to me, I didn't understand why three hashes are needed

Comment (Diaoul) [inline cross/node/digests]: It's not mandatory but it's automatically generated that way.

**Comment by wind-rider** [comment]
Thanks for merging :)

**Comment by Dr-Bean** [comment]
Node 0.12.2-5 has been published on the repository. Thanks @wind-rider.

**Pull request merged**